docs/index.md

```diff
@@ -4,11 +4,18 @@ Bazel rules for building tar, zip, deb, and rpm packages.
 
 Use rules-pkg-discuss@googlegroups.com for discussion.
 
-*   [examples](https://github.com/bazelbuild/rules_pkg/tree/main/examples) -
+*   [Examples](https://github.com/bazelbuild/rules_pkg/tree/main/examples) -
     Cookbook examples for using the rules.
 
 ## Reference
 
-*   [HEAD](https://github.com/bazelbuild/rules_pkg/blob/main/pkg/docs/reference.md)
+We are in the process of migrating from hand-written docs to generated ones.
+Some rules are fully documented in their definitions, and will have complete
+documentation in the Latest or versioned snapshots. Some rules still have
+attributes only defined in in the "Legacy" documentation. You may have to
+consult more than one place to get a complete picture.
+
+*   [Latest Snapshot](latest.md)
```

Comment:
Maybe `latest/reference.md`?  This is fine too.

Reply:
there is no latest/reference.md yet. We could add it, but let's see how it all looks in practice first.